src/server/pfs/server/driver.go

```diff
@@ -1140,7 +1307,9 @@ func (d *driver) makeCommit(
 	if parent == nil {
 		return nil, errors.Errorf("parent cannot be nil")
 	}
-
+	if err := d.validateRepo(txnCtx.Stm, parent.Repo); err != nil {
```

Comment:
I guess we could, although I am not too concerned about the read-only methods. I blocked off commit and branch creation since those two could get us in an unrecoverable state, but I guess it would not be too big of a deal to go ahead and block off the read-only methods as well. I already use a helper function for blocking off the commit and branch creation, so I can just use that.

Reply:
Per our slack conversation, I agree that it's totally fine to leave the read-only methods alone (i.e. don't add any checks for `tombstone`, and don't return any errors if a read-only method is called with a tombstoned repo)

Reply:
One thought: would it make sense to add a check/block to `StartCommit`? In case the user for some reason starts a commit in the deleted repo directly?